helloextend-protection/includes/class-helloextend-protection-orders.php

```
@@ -117,15 +122,60 @@ public function get_product_image_url($product)
+    }
+
+    private function is_item_helloextend_lead($item)
+    {
+        $helloextend_meta = $item->get_meta('_helloextend_data');
+        return $this->is_item_helloextend($item) && (bool) $helloextend_meta['leadToken'];
+    }
+
+    private function get_price_in_cents($item_price)
+    {
+        return (int) floatval($item_price * 100);
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'get_price_in_cents'. (Unused Code Rules)

(UnusedPrivateMethod)